### PR TITLE
Expose misc ops at package level

### PR DESCRIPTION
**Pull request by jdsgomes**
cc @datumbox  @NicolasHug

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

**Review by NicolasHug — COMMENTED** [review]
Thanks for the PR @jdsgomes ,

If we're going to expose these in the `torchvision.ops` namespace, I think we should also expose them in the docs in `ops.rst`. These ops seem to lack docstring though, so maybe we should write some

**Comment by oke-aditya** [comment]
We were going to expose using `torchvision.nn` ? #4333 Or is it decided with `ops.misc.py` ?

**Comment by datumbox** [comment]
@oke-aditya The reason we are not doing it at the moment is because there is no consensus for renaming/moving stuff to `nn`. This PR is just addressing the fact that the classes are public but not exposed directly via `ops`. Though the RFC you send is still valid, I would recommend not to link the two so that we don't block fixing outstanding issues.

**Comment by oke-aditya** [comment]
Can understand it :) thanks for the clarification.

**Comment by jdsgomes** [comment]
> Thanks for the PR @jdsgomes ,
> 
> If we're going to expose these in the `torchvision.ops` namespace, I think we should also expose them in the docs in `ops.rst`. These ops seem to lack docstring though, so maybe we should write some

Do you suggest adding the docstrings in this PR as well or that is out of scope and we should create a ticket for it?

File: torchvision/ops/misc.py [hunk 1]
```diff
@@ -99,12 +102,29 @@ def __repr__(self) -> str:
...
+
+    """
+
     def __init__(
         self,
         in_channels: int,
         out_channels: int,
         kernel_size: int = 3,
-        stride: int = 1,
+        stride: _size_2_t = 1,
```
Comment (datumbox) [inline torchvision/ops/misc.py:128]: If you define stride as `_size_2_t` then probably have kernel_size, padding and dilation like [here](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L419-L422). If you decide to switch back to integers, we will need to also modify the documentation above.

**Review by datumbox — COMMENTED** [review]
Overall looks good. Just one comment on the stride size type. Let me know what you think.

**Pull request merged**